modules/tinymce/src/core/test/ts/webdriver/PageUpDownKeysTest.ts

```diff
@@ -0,0 +1,102 @@
+import { Keys, RealKeys } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('webdriver.tinymce.core.keyboard.PageUpDownKeyTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    add_unload_trigger: false,
+    base_url: '/project/tinymce/js/tinymce',
+    indent: false
+  }, [ Theme ], true);
+  const platform = PlatformDetection.detect();
+
+  // It necessary to add a different assertion for Firefox because it does not move the caret start/end on OS windows and Mac,
+  // Linux Firefox could be an exception, nevertheless we does not override the default behavior.
+  context('Page Up', () => {
+    it('TINY-4612: caret should be placed at the start of the line if the first child is an inline element', async () => {
```

Comment:
This matches what I said in my other comment, but this should **not** be happening on Mac, as it doesn't move the selection when pressing page up/down and as such TinyMCE shouldn't be moving it either.

Reply:
I'll skip assertions on mac and windows firefox, updated